doc/source/theory/multiphysics/electromagnetism/time_harmonic.rst

```diff
@@ -2,4 +2,49 @@
 Time Harmonic Maxwell's Equations
 ==================================
 
-**Under construction**
\ No newline at end of file
+In a future version of Lethe, there will be a module to solve time-harmonic Maxwell's equations which are used in microwave heating processes. The scope of this theory section is to give a brief description of where the equations come from, for further details, the reader is invited to look at the `Griffiths <https://en.wikipedia.org/wiki/Introduction_to_Electrodynamics (book)>`_ book for an introduction on electromagnetism theory and on the following `article <https://ieeexplore.ieee.org/document/9682427>`_ for a review on computational electromagnetism. Thereupon, Maxwell's equations are a set of partial differential equations that form the foundation of classical electromagnetism. This famous set of equations can take multiple forms, but one of the most common ones is the macroscopic volumetric equations in matter :
+
+.. math::
+    \begin{align*}
+    \nabla \cdot \mathbf{D} &= \rho_f \\
+    \nabla \cdot \mathbf{B} &= 0 \\
+    \nabla \times \mathbf{E} &= -\frac{\partial \mathbf{B}}{\partial t} \\
+    \nabla \times \mathbf{H} &= \mathbf{J}_f + \frac{\partial \mathbf{D}}{\partial t}
+    \end{align*}
+
+where:
+
+* :math:`\mathbf{D}` is the electric field flux density;
+* :math:`\mathbf{B}` is the magnetic field flux density;
+* :math:`\mathbf{E}` is the electric field intensity;
+* :math:`\mathbf{H}` is the magnetic field intensity;
+* :math:`\rho_f` is the free electric charge density;
+* :math:`\mathbf{J}_f` is the free current density, and :math:`t` is time.
+
+For linear medium, the electromagnetic fluxes can be related to their intensity with the following constitutive relations and the electric current is proportional to the electric field intensity:
+
+.. math::
+    \begin{align*}
+    \mathbf{D} &= \varepsilon \mathbf{E} \\
+    \mathbf{B} &= \mu \mathbf{H}\\
+    \mathbf{J}_f &= \sigma_e \mathbf{E} + \mathbf{J}_{ext}
+    \end{align*}
+
+where :math:`\varepsilon` is the permittivity, :math:`\mu` is the permeability, :math:`\sigma_e` is the conductivity of the medium and :math:`\mathbf{J}_{ext}` is the externally applied current density. When one is confronted with oscillating electromagnetic fields, a simplification that is commonly made is to consider the fields and exciting currents as time harmonic, `i.e.` they can be expressed as :
```

Comment:
Tricky thing is we always use $\mu$ for dynamic viscosity. We'll have to be careful about that.

Reply:
I could change it for \mu_em for electromagnetic ? Maybe also change \varepsilon to \varepsilon_em then?

Reply:
I changed them, epsilon is also use for the interface in Cahn-Hilliard.